3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,69 @@
+//Flight booking
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (useFormalName == true && gender == "man") {
+    return `Lord ${firstname} ${surname}`;
+  } else if (useFormalName == true && gender == "woman") {
+    return `Miss ${firstname} ${surname}`;
+  } else {
+    return `${firstname} ${surname}`;
+  }
+}
+
+let fullname1 = getFullname("Sevda", "Naziri", true, "woman");
+let fullname2 = getFullname("John", "Wick", true, "man");
+
+console.log(fullname1);
+console.log(fullname2);
+
+//Event application
+let daysUntilEvent = 8;
+
+function getEventWeekday(days) {
+  var date = new Date();
+  date.setDate(date.getDate() + days);
+  return date;
+}
+console.log(getEventWeekday(daysUntilEvent));
+
+//Weather wear
+
+function whatShouldIWear(tempreture) {
+  if (tempreture >= 20) {
+    return `You should wear T-shirt and shorts}`;
+  } else if (tempreture < 20) {
+    return `You must wear warm cloths`;
+  }
+}
+console.log(whatShouldIWear(20));
+
+//Student manager
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (class07Students.length >= 6 && studentName !== "Queen") {
+    return `Cannot add more students to class 07 - Queen always can join the class`;
+  } else if (class07Students.includes(studentName)) {
+    return `Student Benjamin is already in the class`;
+  } else if (studentName === "") {
+    console.log("Cannot add an empty string to a class");
+  } else {
+    class07Students.push(studentName);
+  }
+}
+
+//getNumberOfStudents function
+function getNumberOfStudents() {
+  return class07Students.length;
+}
+addStudentToClass("Benjamin");
+addStudentToClass("Sevda");
+addStudentToClass("Winston");
+addStudentToClass("Theo");
+addStudentToClass("Amalie");
+addStudentToClass("");
+addStudentToClass("Bent");
+addStudentToClass("Peter");
+addStudentToClass("Queen");
+addStudentToClass("Benjamin");
+
+console.log(`Students number ${getNumberOfStudents()}`);
```

Comment:
🤖 You haven't completed the last homework assignment, and I understand that it's marked as optional. However, taking advantage of every opportunity to practice can greatly contribute to your skill development and overall improvement. It's worth considering giving it a try!